examples/react-router-prototyping/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-router-prototyping",
+  "version": "1.0.0",
+  "description": "",
+  "skpm": {
+    "main": "react-router-prototyping.sketchplugin",
+    "manifest": "src/manifest.json"
+  },
+  "scripts": {
+    "build": "skpm-build",
+    "watch": "skpm-build --watch",
+    "render": "skpm-build --watch --run",
+    "render:once": "skpm-build --run",
+    "postinstall": "npm run build && skpm-link"
+  },
+  "author": "Macintosh Helper",
+  "license": "MIT",
+  "devDependencies": {
+    "@skpm/builder": "^0.4.3"
+  },
+  "dependencies": {
+    "prop-types": "^15.5.8",
+    "react": "^16.13.0",
+    "react-router-primitives": "^0.1.2",
```

Comment:
I don't think it's used?

Reply:
Do you mean that `prop-types` isn't used? Should I add prop types across the components?

Reply:
`react-router-primitives` isn't

Reply:
I'm using it in `examples/react-router-prototyping/src/components/AppBar.js`. My thinking was to use `react-sketchapp-router` in the `App.jsx` entry-point, and `react-router-prototyping` in shared component code.